### PR TITLE
Chem fixes, Syringes and Hypovials in Chem master

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
Corazone is now mixable with it's old recipe. The chem master machine (NOT the condimasters) found in chem can now output into syringes that have no custom materials in them, and when upgraded can output into small hypovials. Also, you can alt+click patches to change their appearance because why not.

## Why It's Good For The Game
Fixes are good, more variations are good, and having more options to output from the chemmaster doesn't hurt.

## Changelog
:cl:
fix: Corazone is mixable again with two parts phenol and one part lithium.
fix: Hypovials now show fullness on the icon again.
add: Hypovials are now printable at the medical lathe
add: The chem master can now output into syringes, and when upgraded, can output into hypovials!
rscadd: Patches now have more variations.
/:cl:

### CODE REVIEW

**Comment by MarkSuckerberg** [comment]
ahem

**Comment by Superyodeler** [comment]
*wheeze*

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 1]
```diff
@@ -183,6 +183,11 @@
 	else
 		return 0
 
+/obj/item/reagent_containers/syringe/nomats
+	name = "synthetic syringe"
+	desc = "A syringe that can hold up to 15 units. This one seems to be made out of synthesized materials."
+	custom_materials = null
+
```
Comment (Superyodeler) [inline code/modules/reagents/reagent_containers/syringes.dm:190]: Syringes printed by the chem master start off on draw, and should start on inject.
```suggestion
	mode = SYRINGE_INJECT

```
Also, I think being able to print functionally normal syringes might be too much.

File: waspstation/code/modules/reagents/reagent_containers/patch.dm [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/obj/item/reagent_containers/pill/patch/nicotine
+	name = "nicotine patch"
+	desc = "Helps you stop smoking. Not that it's any healthier."
+	list_reagents = list(/datum/reagent/drug/nicotine = 20)
```
Comment (Superyodeler) [inline waspstation/code/modules/reagents/reagent_containers/patch.dm:4]: Too high, over 15u nicotine causes an overdose, over 10u causes addiction. When testing it, I took 40 toxin damage from one patch.
```suggestion
	list_reagents = list(/datum/reagent/drug/nicotine = 9)
```

File: code/modules/reagents/chemistry/machinery/chem_master.dm [hunk 3]
```diff
@@ -16,6 +16,7 @@
 	var/obj/item/storage/pill_bottle/bottle = null
 	var/mode = 1
 	var/condi = FALSE
+	var/upgraded = FALSE ///Manipulator level >3, allows hypovials
```
Comment (Superyodeler) [inline code/modules/reagents/chemistry/machinery/chem_master.dm:19]: Tried both var editing and upgrading normally, upgraded doesn't appear to have any effect.

Comment (Superyodeler) [reply code/modules/reagents/chemistry/machinery/chem_master.dm:19]: Because of this, you can't dispense into hypovials

Comment (MarkSuckerberg) [reply code/modules/reagents/chemistry/machinery/chem_master.dm:19]: You sure? I tested this and it worked for me

File: code/modules/reagents/reagent_containers/syringes.dm (hunk 1 above)
Comment (MarkSuckerberg) [reply code/modules/reagents/reagent_containers/syringes.dm:190]: Syringes are a dime a dozen, and these ones don't have any materials anyways

File: code/modules/reagents/chemistry/machinery/chem_master.dm (hunk 3 above)
Comment (Superyodeler) [reply code/modules/reagents/chemistry/machinery/chem_master.dm:19]: You're right, I forgot to compile tgui. Seems to be working.

File: code/modules/reagents/reagent_containers/syringes.dm (hunk 1 above)
Comment (Superyodeler) [reply code/modules/reagents/reagent_containers/syringes.dm:191]: 50u beakers are also very common, but the printed bottles are still 30u. This isn't a major concern though.

Comment (triplezeta) [reply code/modules/reagents/reagent_containers/syringes.dm:191]: the alternative version of the chem master, the condimaster, has 50 unit bottles.

**Comment by Superyodeler** [comment]
Only thing I'm really looking for now is margularization

File: code/modules/reagents/chemistry/machinery/chem_master.dm [hunk 4]
```diff
@@ -259,6 +266,10 @@
 			vol_each_max = min(40, vol_each_max)
 		else if (item_type == "bottle")
 			vol_each_max = min(30, vol_each_max)
+		else if (item_type == "syringe")
```
Comment (Superyodeler) [inline code/modules/reagents/chemistry/machinery/chem_master.dm:269]:
```suggestion
		else if (item_type == "syringe")		// Wasp begin - Better chem masters
```

File: code/modules/reagents/chemistry/machinery/chem_master.dm [hunk 5]
```diff
@@ -259,6 +266,10 @@
...
+			vol_each_max = min(15, vol_each_max)
+		else if (item_type == "hypovial")
+			vol_each_max = min(60, vol_each_max)
```
Comment (Superyodeler) [inline code/modules/reagents/chemistry/machinery/chem_master.dm:272]:
```suggestion
			vol_each_max = min(60, vol_each_max)		// Wasp end
```

File: code/modules/reagents/reagent_containers/patch.dm [hunk 6]
```diff
@@ -10,6 +10,16 @@
 	apply_method = "apply"
 	self_delay = 30		// three seconds
 	dissolvable = FALSE
+	unique_reskin = list("plain patch" = "bandaid",
```
Comment (Superyodeler) [inline code/modules/reagents/reagent_containers/patch.dm:13]:
```suggestion
	unique_reskin = list("plain patch" = "bandaid",		// Wasp begin - Patch reskins
```

File: code/modules/reagents/reagent_containers/patch.dm [hunk 7]
```diff
@@ -10,6 +10,16 @@
...
+						"pink patch" = "bandaid_pink",
+						"brute patch" = "bandaid_brute",
+						"burn patch" = "bandaid_burn",
+						"brute/burn patch" = "bandaid_both",
+						"toxin patch" = "bandaid_toxin",
+						"oxy patch" = "bandaid_oxy",
+						"nicotine patch" = "bandaid_smoke",
+						"energy patch" = "bandaid_energy"
+						)
```
Comment (Superyodeler) [inline code/modules/reagents/reagent_containers/patch.dm:22]:
```suggestion
						)		// Wasp end
```

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 8]
```diff
@@ -183,6 +183,12 @@
 	else
 		return 0
 
+/obj/item/reagent_containers/syringe/nomats
+	name = "synthetic syringe"
+	desc = "A syringe that can hold up to 15 units. This one seems to be made out of synthesized materials."
+	custom_materials = null
+	mode = SYRINGE_INJECT
+
```
Comment (Superyodeler) [inline code/modules/reagents/reagent_containers/syringes.dm:191]: This could be moved to a modularized file, or commented out

**Pull request closed without merging**